diff --git a/node_modules/react-native-parallax-scroll-view/src/index.js b/node_modules/react-native-parallax-scroll-view/src/index.js
index 8787b6f..97f0683 100644
--- a/node_modules/react-native-parallax-scroll-view/src/index.js
+++ b/node_modules/react-native-parallax-scroll-view/src/index.js
@@ -1,6 +1,6 @@
 import React, { Component } from 'react'
-import { Animated, Dimensions, View, ViewPropTypes } from 'react-native'
-
+import { Animated, Dimensions, View } from 'react-native'
+import {ViewPropTypes } from 'deprecated-react-native-prop-types'
 const styles = require('./styles')
 
 import { bool, func, number, string } from 'prop-types'
